paasta_tools/utils.py

```diff
@@ -2381,35 +2374,6 @@ def get_synapse_haproxy_url_format(self) -> str:
     def get_service_discovery_providers(self) -> Dict[str, Any]:
         return self.config_dict.get("service_discovery_providers", {})
 
-    def get_cluster_autoscaling_resources(self) -> IdToClusterAutoscalingResourcesDict:
-        return self.config_dict.get("cluster_autoscaling_resources", {})
-
-    def get_cluster_autoscaling_draining_enabled(self) -> bool:
-        """Enable mesos maintenance mode and trigger draining of instances before the
-        autoscaler terminates the instance.
-
-        :returns A bool"""
-        return self.config_dict.get("cluster_autoscaling_draining_enabled", True)
-
-    def get_cluster_autoscaler_max_increase(self) -> float:
-        """Set the maximum increase that the cluster autoscaler can make in each run
-
-        :returns A float"""
-        return self.config_dict.get("cluster_autoscaler_max_increase", 0.2)
-
-    def get_cluster_autoscaler_max_decrease(self) -> float:
-        """Set the maximum decrease that the cluster autoscaler can make in each run
-
-        :returns A float"""
-        return self.config_dict.get("cluster_autoscaler_max_decrease", 0.1)
-
-    def get_cluster_boost_enabled(self) -> bool:
-        """Enable the cluster boost. Note that the boost only applies to the CPUs.
-        If the boost is toggled on here but not configured, it will be transparent.
-
-        :returns A bool: True means cluster boost is enabled."""
-        return self.config_dict.get("cluster_boost_enabled", False)
-
     def get_resource_pool_settings(self) -> PoolToResourcePoolSettingsDict:
         return self.config_dict.get("resource_pool_settings", {})
```

Comment:
i get the feeling that this was also mesos-related: i don't see any hits in sourcegraph for this other than this file: https://sourcegraph.yelpcorp.com/search?q=resource_pool_settings